### PR TITLE
Create 060-ipfs-camp-keynotes-interviews.md

### PR DESCRIPTION
**Pull request by momack2**
Here's a new post combining the keynotes and community interviews! Would love review from @doctorrobinson, @daviddias @parkan and @autonome!

Note, this should launch after https://github.com/ipfs/blog/pull/307

### CODE REVIEW

File: content/post/060-ipfs-camp-keynotes-interviews.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Julian told us about his peer-to-peer collaboration platform Radicle and the benefits of decentralized systems. 
+
+#### [World Brain - Oliver Sauter](https://youtu.be/e1YfSa2_Drc) 
+We talked to Oliver about their decentralized search tool, Memex, and their peer-to-peer search engine for personal and collective knowledge, Worldbrain.io. 
+
+#### [IPFS Force - Steven Li](https://youtu.be/KsMqesFi4Ac) 
+And on top of all that, Steven joined us from Shanghai City to discuss his work on IPFS-Force.
+
+
+## That's All, Folks
```
Comment (doctorrobinson) [inline content/post/060-ipfs-camp-keynotes-interviews.md]: This is great Molly! Read through it all, and it's a pretty dope wrap up to the whole event! 

Should we find a way to host all of Yuni's amazing photos and do a share of those as well?

Comment (doctorrobinson) [reply content/post/060-ipfs-camp-keynotes-interviews.md]: Non-external link here: 
https://drive.google.com/open?id=1vL9tfkWqX1HMNiHhpdvkGVht4H5MjJsE

Comment (momack2) [reply content/post/060-ipfs-camp-keynotes-interviews.md]: I love that idea!! @yunigraham - IIRC you were going to have a selection you wanted to share out. I'm sure folks would love to flip through if we were to upload them somewhere!

**Comment by parkan** [comment]
overall this looks good! small comment re: Juan’s title and how it fits with our positioning of the project as a community effort (maybe “original creator” or something like that?)

File: content/post/060-ipfs-camp-keynotes-interviews.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### Why IPFS? -- Juan Benet
+[![Screen Shot 2019-09-30 at 6 19 45 PM](https://user-images.githubusercontent.com/618519/65927657-6e404b00-e3af-11e9-97f7-4ef38ed31de6.png)](https://www.youtube.com/watch?v=zE_WSLbqqvo&feature=youtu.be)
+
+Juan Benet, the creator of IPFS, used [his keynote on "Why IPFS?"](https://www.youtube.com/watch?v=zE_WSLbqqvo&feature=youtu.be) to help everyone zoom out to the wider context around our moment in time, how technology is evolving, and how IPFS fits into the bigger picture. He described the history of the internet, some potential predictions about the future, and how IPFS can address some major problems in the web as it exists today (notably censorship, inefficiency, and poor offline usability). Persistence, innovation, and determination are required to ensure we continue making progress for the long haul.
```
Comment (momack2) [inline content/post/060-ipfs-camp-keynotes-interviews.md]:
```suggestion
Juan Benet, the original creator of IPFS, used [his keynote on "Why IPFS?"](https://www.youtube.com/watch?v=zE_WSLbqqvo&feature=youtu.be) to help everyone zoom out to the wider context around our moment in time, how technology is evolving, and how IPFS fits into the bigger picture. He described the history of the internet, some potential predictions about the future, and how IPFS can address some major problems in the web as it exists today (notably censorship, inefficiency, and poor offline usability). Persistence, innovation, and determination are required to ensure we continue making progress for the long haul.
```

File: content/post/060-ipfs-camp-keynotes-interviews.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+[![Screen Shot 2019-09-30 at 6 19 33 PM](https://user-images.githubusercontent.com/618519/65927659-700a0e80-e3af-11e9-9282-5c405c4ace1c.png)](https://www.youtube.com/watch?v=jpQnQbfhuBc&feature=youtu.be)
+
+Want to learn about how the IPFS Community and Ecosystem have been growing? [This video](https://www.youtube.com/watch?v=jpQnQbfhuBc&feature=youtu.be) from Molly Mackinlay, IPFS Project Lead, is a great overview of the scale, progress, and important milestones IPFS has achieved so far in 2019.
+
+### Closing
+On our last day, we celebrated the amazing learning, presentations, and courses with a [Space Training Program Graduation & Awards Ceremony](https://www.youtube.com/watch?v=INVQPepMMMU&feature=youtu.be). Who got the award for best interactive skit? Watch this to see the team!
+
+And finally, Juan set us back out through the airlock back toward Earth with his [IPFS Camp Sendoff](https://youtu.be/HDc7kEYHang), including a huge **THANK YOU** to everyone who made this camp possible - especially the tireless Protocol Labs Events Team who worked so hard to make this event a huge success!
+
+## Community Interviews
```
Comment (momack2) [inline content/post/060-ipfs-camp-keynotes-interviews.md]:
```suggestion
### Recap
Curious how everything fit together, or want to understand how the pieces of IPFS Camp became a greater whole? Check out [this overview from Molly](https://youtu.be/cQJXnJDbXv4) on how different components like the Poster Sessions & Deep Dives, SciFi Fair, Lightning Talks, and of course the Core & Elective Courses, wove together into a great community event. Many connections between different attendees and projects emerged from IPFS Camp too - spurring new collaborations and integrations. These great recaps from [Textile](https://medium.com/textileio/ipfs-camp-2019-the-highlights-and-takeaways-2b3cb4f42513), [Berty](https://berty.tech/blog/ipfs-camp/), and of course [the IPFS blog](https://blog.ipfs.io/2019-07-08-ipfs-camp-recap/) highlight a few!

## Community Interviews
```

File: content/post/060-ipfs-camp-keynotes-interviews.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+[![Screen Shot 2019-09-30 at 6 19 33 PM](https://user-images.githubusercontent.com/618519/65927659-700a0e80-e3af-11e9-9282-5c405c4ace1c.png)](https://www.youtube.com/watch?v=jpQnQbfhuBc&feature=youtu.be)
+
+Want to learn about how the IPFS Community and Ecosystem have been growing? [This video](https://www.youtube.com/watch?v=jpQnQbfhuBc&feature=youtu.be) from Molly Mackinlay, IPFS Project Lead, is a great overview of the scale, progress, and important milestones IPFS has achieved so far in 2019.
+
+### Closing
+On our last day, we celebrated the amazing learning, presentations, and courses with a [Space Training Program Graduation & Awards Ceremony](https://www.youtube.com/watch?v=INVQPepMMMU&feature=youtu.be). Who got the award for best interactive skit? Watch this to see the team!
+
+And finally, Juan set us back out through the airlock back toward Earth with his [IPFS Camp Sendoff](https://youtu.be/HDc7kEYHang), including a huge **THANK YOU** to everyone who made this camp possible - especially the tireless Protocol Labs Events Team who worked so hard to make this event a huge success!
+
+### Recap
```
Comment (momack2) [inline content/post/060-ipfs-camp-keynotes-interviews.md]:
```suggestion
### Recap
![image](https://user-images.githubusercontent.com/618519/66095571-d7a99080-e54c-11e9-9db7-906fe034f888.png)

```

**Comment by momack2** [comment]
My aim is for this to roll out on Monday Japan time right before DevCon. Would love any remaining reviews!

**Comment by doctorrobinson** [comment]
Once you ship I'll flip all the videos + playlists live

File: content/post/060-ipfs-camp-keynotes-interviews.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+date: 2019-10-14
+url: 2019-10-14-ipfs-camp-keynotes-interviews
+title: IPFS Camp Keynotes and Interviews
+author: Molly Mackinlay
+---
+
+If you've been following the many IPFS Camp updates here in the blog, you've likely already explored the videos from our [lightning talks](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks), [deep dives](https://github.com/ipfs/camp#-deep-dives), [poster projects](https://github.com/ipfs/camp#-poster-projects), [core and elective courses](https://github.com/ipfs/camp#-core--elective-courses), and most recently [the SciFi Fair](https://github.com/ipfs/camp#-sci-fi-fair). Today, we're excited to share the Keynotes that brought us together on our first and last days of IPFS Camp, and the project interviews that highlight some of the amazing work being done by IPFS Camp attendees in the wider ecosystem.
```
Comment (momack2) [inline content/post/060-ipfs-camp-keynotes-interviews.md]:
```suggestion
If you've been following the many IPFS Camp updates here in the blog, you've likely already explored the videos from our [lightning talks](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks), [deep dives](https://github.com/ipfs/camp#-deep-dives), [poster projects](https://github.com/ipfs/camp#-poster-projects), [core and elective courses](https://github.com/ipfs/camp#-core--elective-courses), and most recently [the SciFi Fair](https://github.com/ipfs/camp#-sci-fi-fair). Today, we're excited to share the **Keynotes** that brought us together on our first and last days of IPFS Camp, and the **Community Interviews** that highlight some of the amazing work being done by IPFS Camp attendees in the wider ecosystem.
```

**Comment by momack2** [comment]
merging this to roll out on monday

**Comment by momack2** [comment]
@olizilla @autonome - this was submitted, the date has passed, and this isn't visible in the blog yet. can haz help? Is there a thing to kick somewhere?

**Comment by autonome** [comment]
I've rerun the job from here:

https://circleci.com/gh/ipfs/workflows/blog/tree/master

**Comment by autonome** [comment]
And now I'm getting permissions error, so it's not strictly a permissions issue.

The infra ticket is here: https://github.com/ipfs/infra/issues/492

**Comment by olizilla** [comment]
This is frustrating. We are all on a GitHub team with permissions on the ci context, so this should work, all seemless like before, but no, it's the worst. I will kick it this eve and fix it this week.

**Comment by autonome** [comment]
Looks like there's a build error now. Builds fine locally tho.

>  Error pulling image circleci/node:10.15.1-browsers: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying
>  image cache not found on this host, downloading circleci/node:10.15.1-browsers

**Comment by doctorrobinson** [comment]
I flipped videos from unlisted to live in anticipation of this. Once this post is up, ALL ipfs camp content will have been distributed.

And now to get it all on Dtube...

**Comment by olizilla** [comment]
the blog post is a thing https://blog.ipfs.io/2019-10-14-ipfs-camp-keynotes-interviews/

I hit "rebuild from start" and it did just that.

> Error pulling image circleci/node:10.15.1-browsers: Error response from daemon: received unexpected HTTP status: 503 Service Unavailable... retrying

suggests that the notoriusly semi-reliable _dockerhub_ was unavailable when you restarted the build @autonome 

I will add this to the list of reasons why we should not rely on dockerhub for anything.

**Comment by momack2** [comment]
Thank you, Oli!

**Comment by doctorrobinson** [comment]
Whoot whoot! Thanks everyone!

**Comment by autonome** [comment]
> I will add this to the list of reasons why we should not rely on dockerhub for anything.

Can you file an issue here in the blog repo that describes what we should do instead?

**Comment by olizilla** [comment]
Is not so simple. This is more of a plantive wail... circle-ci uses docker images for just about everything, and by default they are pulled from dockerhub. We'd need to look at using an alternative registry for the ipfs-dns-deploy stuff. But dockerhub is just reliable _enough_ that it remains a low priority

**Comment by autonome** [comment]
Do you think this is related to the intermittent permissions problem?

**Comment by olizilla** [comment]
no, that's the super fun thing here; it's two unrelated issues _at the same time_!!

1. circle ci permissions weirdness, where by it doesn't correctly authenticate users who are in approved github teams (seems to be happening regularly, though our config appears to be correct)
2. random failure of a ci build as it couldnt fetch the build image from docker hub. (happens once every few months)

**Comment by autonome** [comment]
You'd make a great doctor, if only for having the *best* bedside manner.

**Comment by olizilla** [comment]
@momack2 can you check to ensure you are following the ipfs/blog project on circleci... go here https://circleci.com/gh/ipfs/workflows/blog/tree/master and see if there is a "follow project" button somewhere... i think top right. Please do follow it if you are not already.

**Comment by momack2** [comment]
I had to sign in and authorize circleci to view all my private
repositories. I also found a "follow project" button in the settings menu.

**Pull request merged**